src/test/regress/sql/multi_extension.sql

```diff
@@ -175,3 +175,131 @@ ALTER EXTENSION citus UPDATE;
 
 -- if cache is invalidated succesfull, this \d should work without any problem
 \d
+
+\c - - - :master_port
+
+CREATE VIEW table_fkeys AS
+SELECT r.conname AS "Constraint",
+       pg_catalog.pg_get_constraintdef(r.oid, true) AS "Definition",
+       conrelid AS "relid"
+FROM pg_catalog.pg_constraint r
+WHERE r.contype = 'f';
```

Comment:
Actually, the problem I'm having is I can't find the referenced column in a foreign key created against a column which already has a unique index. The following isn't 100% complete (it's missing schema declarations), but it's enough to demonstrate:

```sql
CREATE TABLE mx_test_schema_1.mx_table_1 (col1 int UNIQUE, col2 text);
CREATE TABLE mx_test_schema_2.mx_table_2 (col1 int, col2 text);
CREATE UNIQUE INDEX mx_index_4 ON mx_test_schema_2.mx_table_2(col1);

ALTER TABLE 
	mx_test_schema_1.mx_table_1 
ADD CONSTRAINT 
	mx_fk_constraint 
FOREIGN KEY 
	(col1)
REFERENCES
	mx_test_schema_2.mx_table_2(col1);

SELECT * FROM information_schema.referential_constraints;
--  constraint_catalog | constraint_schema |     constraint_name      | unique_constraint_catalog | unique_constraint_schema |   unique_constraint_name    | match_option | update_rule | delete_rule 
-- --------------------+-------------------+--------------------------+---------------------------+--------------------------+-----------------------------+--------------+-------------+-------------
--  regression         | mx_test_schema_1  | mx_fk_constraint         |                           |                          |                             | NONE         | NO ACTION   | NO ACTION

```